readme.md

```diff
@@ -85,7 +85,7 @@ distribution {
 }
 ```
 
-sls-packaging also maintains a lockfile, `product-dependencies.lock`, which should be checked in to Git.  This file is an accurate reflection of all the inferred and explicitly defined product dependencies. Run **`./gradlew --write-locks`** to update it. e.g.
+sls-packaging also maintains a lockfile, `product-dependencies.lock`, which should be checked in to Git.  This file is an accurate reflection of all the inferred and explicitly defined product dependencies. Run **`./gradlew --write-locks`** or **`./gradlew writeProductDependenciesLocks`** to update it. e.g.
```

Comment:
There are also assorted error messages prompting the user to run `--write-locks` (and the comment at the top of the `product-dependencies.lock` files). I didn't change those, since most users likely won't benefit much from this marker task.